md/database-configuration.md

```diff
@@ -1,285 +1,108 @@
-# Database configuration for engine data
+# Database creation and settings to work with Bonita BPM 
 
 ## Database usage
 
-#### Introduction
+Bonita BPM uses an RDBMS (Relational DataBase Management System) for the following purposes:
+ - One database schema is required by Bonita BPM Engine to store information about deployed process definitions, process configurations, history of process execution, users, as well as Bonita BPM Platform configuration information. 
+- We recommend that you configure a different database schema if your project uses [business data](define-and-deploy-the-bdm.md).
 
-The Bonita BPM Engine uses an RDBMS (Relational DataBase Management System) to store information about deployed process definitions, process configurations, history of process execution, users, etc.  
-One database schema is required by Bonita BPM Engine to store all information.
+Bonita BPM supports MySQL, PostgreSQL, SQL Server, and Oracle RDBMSs.  
+For Subscription users, you can find the updated list of RDBMSs versions supported in the [Customer Portal supported RDBMSs](https://customer.bonitasoft.com/support-policies/2219).
```

Comment:
Je suggère:
Bonita BPM supports MySQL, PostgreSQL, SQL Server, and Oracle RDBMSs in the [following versions](https://customer.bonitasoft.com/system/files/customer_portal/ServerEnvMatrix_7.png).
Comme ça même nos utilisateurs communautaires peuvent connaître les version des SGBD qu'on supporte, c'est la moindre des choses.